docker-compose/docker-compose.multiserver.build.yml

```diff
@@ -0,0 +1,134 @@
+# Copyright 2016 The WWU eLectures Team All rights reserved.
+#
+# Licensed under the Educational Community License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#     http://opensource.org/licenses/ECL-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+volumes:
+  data: {}
+  db: {}
+
+services:
+  mariadb:
+    image: mariadb:10.0
+    environment:
+      - MYSQL_ROOT_PASSWORD=root
+      - MYSQL_DATABASE=opencast
+      - MYSQL_USER=opencast
+      - MYSQL_PASSWORD=opencast
+    command: '--wait_timeout=28800'
+    volumes:
+      - ./assets/opencast-ddl.sql:/docker-entrypoint-initdb.d/opencast-ddl.sql:ro
+      - db:/var/lib/mysql
+
+  activemq:
+    image: webcenter/activemq:5.14.3
+    environment:
+      - ACTIVEMQ_MIN_MEMORY=128
+      - ACTIVEMQ_MAX_MEMORY=1024
+      - ACTIVEMQ_ENABLED_SCHEDULER=true
+      - ACTIVEMQ_REMOVE_DEFAULT_ACCOUNT=true
+      - ACTIVEMQ_OWNER_LOGIN=admin
+      - ACTIVEMQ_OWNER_PASSWORD=password
+    volumes:
+      - ./assets/activemq.xml:/opt/activemq/conf/activemq.xml:ro
+
+  admin:
+    image: quay.io/opencast/build:8.3
+    tty: true
+    stdin_open: true
+    environment:
+      - OPENCAST_BUILD_USER_UID=${OPENCAST_BUILD_USER_UID:-1000}
+      - OPENCAST_BUILD_USER_GID=${OPENCAST_BUILD_USER_GID:-1000}
+      - ORG_OPENCASTPROJECT_SERVER_URL=http://opencast-admin:8080
```

Comment:
Is that working for you? Normally, this should correspond to the service name which is just `admin` in this case. The other compose files use `opencast-admin` so I would do the same here. See also `presentation` and `worker`.